meta-resin-common/recipes-connectivity/modemmanager/modemmanager_1.10.0.bb

```diff
@@ -9,27 +9,32 @@ LIC_FILES_CHKSUM = " \
 
 inherit gnomebase gettext systemd vala gobject-introspection bash-completion
 
-DEPENDS = "glib-2.0 libgudev dbus-glib intltool-native"
+DEPENDS = "glib-2.0 libgudev dbus-glib intltool-native libxslt-native"
 
 SRC_URI = "http://www.freedesktop.org/software/ModemManager/ModemManager-${PV}.tar.xz \
-           file://0001-Add-configure-check-for-canonicalize_file_name-befor.patch \
+           file://0001-Do-not-set-Wno-unused-but-set-variable.patch \
            "
 SRC_URI[md5sum] = "92d25176d0cc9d588ca29700b33c7d98"
 SRC_URI[sha256sum] = "fd0f39996025dac96995daea8a58ec7dd571582f7563a8ae0d5f65b571b76ee2"
```

Comment:
The SRC_URI is the same? And we did an update where we removed a patch? I don't follow

Reply:
This syncs with the upstream. To get this kind of changes you need to follow their history. For this specific one: https://github.com/openembedded/meta-openembedded/commit/42ab6b30ec52a12a6b9ce1c661d126a66b9ded9c#diff-44d914087ecadf8b04045530d71b9533